x-pack/plugins/monitoring/common/http_api/elasticsearch_settings/get_elasticsearch_settings_cluster.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+
+export const getElasticsearchSettingsClusterResponsePayloadRT = rt.type({
+  // TODO: add payload entries
```

Comment:
is the payload here of the same type as we have on this test file? https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/server/lib/elasticsearch_settings/cluster.test.ts#L55

Reply:
No, this is meant as a placeholder for when we convert the client side to TS as well. Sorry that I didn't point that out.